src/cli/extensions/helpers.js

```diff
@@ -109,6 +117,8 @@ function init(roc, state) {
 }
 
 function checkDependencies(roc, state) {
+    debug('Running checkDependencies.');
```

Comment:
I think it will be more informational if the messages states what the code will do and not what function that will run.